framework/src/Volo.Abp.ExceptionHandling/Volo/Abp/AspNetCore/ExceptionHandling/DefaultExceptionToErrorInfoConverter.cs

```diff
@@ -326,7 +329,11 @@ protected virtual AbpExceptionHandlingOptions CreateDefaultOptions()
         return new AbpExceptionHandlingOptions
         {
             SendExceptionsDetailsToClients = false,
-            SendStackTraceToClients = true
+            SendStackTraceToClients = true,
+            SendExceptionDataToClientTypes =
+            {
+                typeof(IBusinessException)
+            }
```

Comment:
We should also pass the `SendExceptionDataToClientTypes` parameter to the [Convert](https://github.com/abpframework/abp/blob/9c1c3a45f896e0d43a211c670216eeaf5af8916b/framework/src/Volo.Abp.ExceptionHandling/Volo/Abp/AspNetCore/ExceptionHandling/DefaultExceptionToErrorInfoConverter.cs#L57) method calls. Otherwise, we can't configure the default option as intended, since CreateDefaultOptions method only adds the `IBusinessException` interface.

Current services that use this option: `AbpExceptionHandlingMiddleware`, `UserExceptionInformer`, `AbpExceptionFilter`, `AbpExceptionPageFilter`, and `ErrorController`.

Reply:
https://github.com/abpframework/abp/pull/20812/commits/bdbe95bc79907b38c8d978086d4242172b8967cf